2-mandatory/1-weather-report.js

```diff
@@ -11,25 +11,29 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
-}
-
+    function getTemperatureReport(cities){
+        const statement = [];
+        for(const city of cities){
+        const temperature = temperatureService(city)
+        statement.push(`The temperature in ${city} is ${temperature} degrees`)
+        }
+        return statement;
+    }
```

Comment:
Comment: Well done!

Consider: You could also iterate [i]
function getTemperatureReport(cities) {
const temperatureReport = [];
    for(let i = 0; i < cities.length; i++) {
        const temperature = temperatureService(cities[i]);
        temperatureReport.push(`The temperature in ${cities[i]} is ${temperature} degrees`);
    }
    return temperatureReport;